front-end/my-vite-app/src/Survey.jsx

```diff
@@ -86,17 +130,27 @@ const Survey = ({closeModal}) => {
                 </div>
             </div>}
             {step === 4 && <div>
-                <h2>What Languages are you Interested in</h2>
-                <button onClick = {() => setLanguages('Python')}>Python</button>
-                <button onClick = {() => setLanguages('Java')}>Java</button>
-                <button onClick = {() => setLanguages('JavaScript')}>JavaScript</button>
-                <button onClick = {() => setLanguages('Ruby')}>Ruby</button>
-                <button onClick = {() => setLanguages('C')}>C</button>
+                <h3>What Languages are you Interested in</h3>
+                <div className = "container">
+                    <button onClick = {() => handleLanguageSelection('Python')}>Python</button>
+                    <button onClick = {() => handleLanguageSelection('Java')}>Java</button>
+                    <button onClick = {() => handleLanguageSelection('JavaScript')}>JavaScript</button>
+                    <button onClick = {() => handleLanguageSelection('Ruby')}>Ruby</button>
+                    <button onClick = {() => handleLanguageSelection('C')}>C</button>
+                </div>
```

Comment:
Can you please look at the comments re. how to refactor this to look cleaner? I left them on an older diff: https://github.com/rubyoliveira/Code4All/commit/da22e3ae0e098db3951eaec527af3a7fc7e38d73